app/views/key_people_forms/new.html.erb

```diff
@@ -1,13 +1,18 @@
 <%= render("shared/back", back_path: back_key_people_forms_path(@key_people_form.reg_identifier)) %>
 
-<div class="text">
+<% if @key_people_form.maximum_key_people == 1 %>
```

Comment:
We make reference to `@key_people_form.maximum_key_people == 1` in a number of places in this view, and I think we could better express what we want with a method call.

You also created `can_have_multiple_key_people?` so perhaps `unless can_have_multiple_key_people?` or something new like `only_one_person_allowed?` (said I haven't got my awake head on 😩, sorry)

Reply:
That should have been replaced with `can_have_multiple_key_people?`. Will sort it out 👍

Reply:
Flipped this around to `can_only_have_one_key_person?` in the end as we were using the inverse of `can_have_multiple_key_people?` more often than not.